host_vars/fffw-lebenshilfe-nno-ap-2ghz/base.yml

```diff
@@ -0,0 +1,11 @@
+---
+location: fffw-lebenshilfe
+role: ap
+target: "ath79/generic"
+model: "mikrotik_sxtsq-2-lite"
+
+wireless_profile: freifunk_fw
+
+mac_override:
+  eth0: cc:2d:e0:9c:4d:58
+...
```

Comment:
?

Reply:
I've saw that configuration on a couple of locations using the Mikrotik-SXTsq5ac-antennas. I thought that it needs to be that way. Does OpenWrt get the correct MAC-Address on those devices by itself?

Reply:
No the comment is because of L11

Reply:
oh, that one. I 've read somewhere that it's not strictly necessary, but good practice to end a YAML-File with three dots. Like it was begun with three dashes.

Reply:
I think that one has been resolved.